force-app/main/default/flexipages/GraphQL.flexipage-meta.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<FlexiPage xmlns="http://soap.sforce.com/2006/04/metadata">
+    <flexiPageRegions>
+        <name>region1</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <itemInstances>
+            <componentInstance>
+                <identifier>graphqlContacts</identifier>
+<componentName>graphqlContacts</componentName>
+            </componentInstance>
+        </itemInstances>
+        <name>region2</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <itemInstances>
+            <componentInstance>
+                <identifier>graphqlVariables</identifier>
+<componentName>graphqlVariables</componentName>
+            </componentInstance>
+        </itemInstances>
+        <name>region3</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <itemInstances>
+            <componentInstance>
+                <identifier>graphqlPagination</identifier>
+<componentName>graphqlPagination</componentName>
+            </componentInstance>
+        </itemInstances>
+        <name>region4</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <masterLabel>GraphQL</masterLabel>
```

Comment:
Let's indicate that this feature is Beta for now:
```suggestion
    <masterLabel>GraphQL (Beta)</masterLabel>
```

Reply:
I updated this, but it appears to actually be the tab label that gets displayed.